src/main/steps/pay-your-fee/content.ts

```diff
@@ -0,0 +1,42 @@
+import { TranslationFn } from '../../app/controller/GetController';
+import { FormContent } from '../../app/form/Form';
+import { CommonContent } from '../common/common.content';
+
+const en = ({ isDivorce, partner, wantToClaimCosts }: CommonContent) => ({
+  title: `Pay your ${isDivorce ? 'divorce' : 'ending your civil partnership'} fee`,
+  line1: `The ${
+    isDivorce ? 'divorce' : 'ending your civil partnership'
+  } application fee is £550. Your application will not be submitted to the court until you have paid.`,
```

Comment:
I reckon we should leave the £550 in for now and change it when we integrate with the fees register